src/systems/tracked-controls.js

```diff
@@ -7,6 +7,7 @@ var utils = require('../utils');
  * It maintains a list with the available tracked controllers
  */
 module.exports.System = registerSystem('tracked-controls', {
+  schema: {alwaysAllowIdPrefixes: {type: 'array'}},
```

Comment:
the `pose` is not part of the gamepad spec. It's an extension specific to VR. All tracked controllers should have a pose attribute even if that pose is not valid yet. for that we can check the hasOrientation / hasPosition flags